cloudimg/ms_azure.py

```diff
@@ -375,6 +379,15 @@ def upload_to_container(self, image_path, container_name, object_name,
             create=True
         )
 
+        # Azure can't handle compressed images on marketplaces so we need to
+        # send the decompressed data to its storage account.
+        if object_name.endswith(".xz"):
+            log.info("Processing a LZMA compressed file.")
```

Comment:
Perhaps the log message should include the name?  A single run can process many files, right?  Is any of it parallelized?  That'd make it ambiguous which log message refers to which file.

Reply:
Sure, I'll include the name, thanks!